presto-native-execution/scripts/release-centos-dockerfile/Dockerfile

```diff
@@ -0,0 +1,162 @@
+# prestissimo
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#============================================================
+# /////////////////////////////////
+#    BUILD TASK DEFINITION
+# /////////////////////////////////
+#============================================================
+ARG BASE_IMAGE=quay.io/centos/centos:stream8
+FROM ${BASE_IMAGE} AS build-stage
+
+ARG CPU_TARGET=avx
+ARG PRESTODB_REPOSITORY=https://github.com/prestodb/presto
+ARG PRESTODB_CHECKOUT=origin/master
+
+USER root:root
+
+ENV CPU_TARGET=${CPU_TARGET}
+ENV TZ=Europe/Warsaw
+
+ENV CC=/opt/rh/gcc-toolset-9/root/bin/gcc
+ENV CXX=/opt/rh/gcc-toolset-9/root/bin/g++
+
+ENV PRESTODB_HOME=/opt/presto
+ENV DEPENDENCY_DIR=/opt/dependency
+ENV INSTALL_PREFIX=/usr/local
+ENV CMAKE_PREFIX_PATH=/usr/local
+ENV PROMPT_ALWAYS_RESPOND=n
+
+ENV PATH=${DEPENDENCY_DIR}/install/bin:${DEPENDENCY_DIR}/install/bin/hadoop-2.10.1/bin:$PATH
+
+ENV PRESTO_ENABLE_PARQUET=ON
+ENV PRESTO_ENABLE_S3=ON
+ENV PRESTO_ENABLE_HDFS=ON
+ENV PRESTO_ENABLE_TESTING=ON
+ENV CLEANUP_BUILD_ARTIFACTS=1
+
+SHELL ["/bin/bash", "-c"]
+#-----------------------------------------------------------
+#---( START of PrestoDB and Velox dependency tree build scripts
+#-----------------------------------------------------------
+WORKDIR ${PRESTODB_HOME}
+RUN set -exu && \
+    dnf -y install git wget unzip pkg-config python3-devel java clang-tools-extra bzip2 bzip2-libs && \
```

Comment:
Convention in Velox so far has been to have fairly light weight dockerfiles , which call into scripts to do the heavy lifting regarding installation of dependencies etc. This is because its easier to validate/remove these dependencies from these script files .

Reply:
True. Fixed.

Reply:
I have only left the minimum required in case the source image changes.